hbase-server/src/test/java/org/apache/hadoop/hbase/compactionserver/TestCompactionServer.java

```diff
@@ -277,20 +289,20 @@ public void testCompactionOffloadTableDescriptor() throws Exception {
 
     TableDescriptor htd =
         TableDescriptorBuilder.newBuilder(TEST_UTIL.getAdmin().getDescriptor(TABLENAME))
-            .setCompactionOffloadEnabled(true).build();
+            .setCompactionOffloadEnabled(false).build();
     TEST_UTIL.getAdmin().modifyTable(htd);
     TEST_UTIL.waitUntilAllRegionsAssigned(TABLENAME);
     // invoke compact
     TEST_UTIL.compact(TABLENAME, false);
-    TEST_UTIL.waitFor(6000, () -> COMPACTION_SERVER.requestCount.sum() > 0);
-    long requestCount = COMPACTION_SERVER.requestCount.sum();
+    Thread.sleep(1000);
```

Comment:
Mind explaining a bit here?

Reply:
OK,this sleep is no need. My Initially thought is compact request is async, this sleep will make the check below effective, After view code I find the increment of request count on compaction server is sync in compaction request